pkg/flow/internal/controller/node_builtin_component.go

```diff
@@ -73,12 +73,12 @@ type ComponentGlobals struct {
 	GetServiceData      func(name string) (interface{}, error) // Get data for a service.
 }
 
-// ComponentNode is a controller node which manages a user-defined component.
+// BuiltinComponentNode is a controller node which manages a user-defined component.
```

Comment:
"user-defined component" is ambiguous now that we're going to introduce the concepts of custom components. 

```suggestion
// BuiltinComponentNode is a controller node which manages a builtin component.
```